scripts/uplift-merges.sh

```diff
@@ -60,7 +60,7 @@ echo "Label: \"$label\""
 echo ""
 
 # Fetch the uplift commits from the GitHub repository
-json_data=$(gh pr list --repo "$repo" --label "$label" --state closed --json "mergedAt,mergeCommit,number,url")
+json_data=$(gh pr list --repo "$repo" --label "$label" --state merged --json "mergedAt,mergeCommit,number,url,title" | jq -c .)
```

Comment:
You had `closed`, any specific reason? merged seems more useful since it would skip PRs that were closed before they were merged.